Makefile

```
@@ -175,16 +178,30 @@ easy-rsa: easy-rsa.tar.gz
-		chmod +x cfssl; \
+	if curl --output /dev/null --silent --head --fail $(CFSSL_URL); then \
+		echo "URL exists; Pulling prebuilt cfssl."; \
+		curl --retry 10 -L -o cfssljson $(CFSSL_URL); \
+		chmod +x cfssljson; \
+	else \
+		echo "URL does not exist; Building cfssljson."; \
+		GOBIN=`pwd` go install github.com/cloudflare/cfssl/cmd/...@v$(CFSSL_VERSION); \
+	fi \
 	fi
 
 cfssljson:
```

checkmake (0.3.2)

[warning] Target body for "cfssljson" exceeds allowed length of 5 lines (10).

(maxbodylength)